pkgs/tools/networking/network-manager/hardcode-gsettings.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/src/wireless-security/eap-method.c b/src/wireless-security/eap-method.c
+index 2e9daa23..940c1d6d 100644
+--- a/src/wireless-security/eap-method.c
++++ b/src/wireless-security/eap-method.c
+@@ -265,7 +265,10 @@ eap_method_ca_cert_ignore_get (EAPMethod *method, NMConnection *connection)
+ static GSettings *
+ _get_ca_ignore_settings (NMConnection *connection)
+ {
++	GSettingsSchemaSource *schema_source;
```

Comment:
Maybe we should cherry-pick https://gitlab.gnome.org/GNOME/glib/merge_requests/749 or unref it as in e1094c3408096d0b22d2e33c423c560942dccc9f

Reply:
I'm going with https://github.com/NixOS/nixpkgs/commit/e1094c3408096d0b22d2e33c423c560942dccc9f